From: Johannes Berg <johannes.berg@intel.com>
Date: Wed, 12 May 2021 15:12:27 +0200
Subject: [NOUPSTREAM] mac80211: include <linux/kcov.h>

The include of kcov.h was removed from sched.h and upstream
updated the includes here in this exact way, but pull that
into our tree now so we can work against 5.13-rc1.

type=cleanup
ticket=none

Change-Id: Icb87f88a9f416c2f04f9ff5186bc3fad6eef02da
Signed-off-by: Johannes Berg <johannes.berg@intel.com>
Reviewed-on: https://git-amr-3.devtools.intel.com/gerrit/317466
automatic-review: ec ger unix iil jenkins <EC.GER.UNIX.IIL.JENKINS@INTEL.COM>
Tested-by: ec ger unix iil jenkins <EC.GER.UNIX.IIL.JENKINS@INTEL.COM>
x-iwlwifi-stack-dev: f155937a1e046c3ffff2620994883486e995b89f
(backported from commit 5551a3b442860fb4bd0fc34eb2136cfd43bd112f
Ignore changes in versions)
Signed-off-by: You-Sheng Yang <vicamo.yang@canonical.com>
---
 net/mac80211/iface.c | 1 +
 net/mac80211/rx.c    | 1 +
 2 files changed, 2 insertions(+)

diff --git a/net/mac80211/iface.c b/net/mac80211/iface.c
index ad44184..7695391 100644
--- a/net/mac80211/iface.c
+++ b/net/mac80211/iface.c
@@ -15,6 +15,7 @@
 #include <linux/if_arp.h>
 #include <linux/netdevice.h>
 #include <linux/rtnetlink.h>
+#include <linux/kcov.h>
 #include <net/mac80211.h>
 #include <net/ieee80211_radiotap.h>
 #include "ieee80211_i.h"
diff --git a/net/mac80211/rx.c b/net/mac80211/rx.c
index f641030..b0762d3 100644
--- a/net/mac80211/rx.c
+++ b/net/mac80211/rx.c
@@ -17,6 +17,7 @@
 #include <linux/etherdevice.h>
 #include <linux/rcupdate.h>
 #include <linux/export.h>
+#include <linux/kcov.h>
 #include <linux/bitops.h>
 #include <net/mac80211.h>
 #include <net/ieee80211_radiotap.h>
